verify/verify-pylint.sh

```diff
@@ -20,11 +20,4 @@ set -o pipefail
 export PYLINTHOME=$(mktemp -d)
 export PATH="${TEST_SRCDIR}/pylint:${PATH}"
 
-# TODO(fejta): all python files
-pylint scenarios/*.py
-pylint jenkins/bootstrap.py
-pylint jenkins/janitor.py
-pylint jobs/*.py
-pylint queue-health/graph/graph.py
-pylint queue-health/weekly_commit_stats.py
-pylint triage/*.py
+find . -type f -name *.py -not -path "./external*" -not -path "./gubernator/third_party*" -print | xargs pylint
```

Comment:
the bazel runfiles tree is all symlinks. to use `-type f` you need to do `find -L . -type f ...`